rollup.config.js

```diff
@@ -1,20 +1,31 @@
 import typescript from "rollup-plugin-typescript2";
+import { uglify } from "rollup-plugin-uglify";
 
-export default {
-  input: "./src/index.ts",
+export default [
+  {
+    input: "./src/index.ts",
 
-  plugins: [typescript()],
+    plugins: [typescript(), uglify()],
 
-  output: [
-    {
-      file: "dist/vue-simpleicons.umd.js",
-      format: "umd",
-      name: "vueSimpleIcons"
-    },
-    {
-      file: "dist/vue-simpleicons.esm.js",
-      format: "esm",
-      name: "vueSimpleIcons"
-    }
-  ]
-};
+    output: [
+      {
+        file: "dist/vue-simpleicons.umd.js",
+        format: "umd",
+        name: "vueSimpleIcons"
+      }
+    ]
+  },
+  {
+    input: "./src/index.ts",
+
+    plugins: [typescript()],
```

Comment:
shouldn't esm be minified too?

Reply:
> Ideally we also minify the esm output, but I didn't get that working with the plugin that I used (rolllup-plugin-uglify) quickly. Any help in this regard @sh7dm ?

Tested again and it still doesn't work out of the box, here is the error output I get on the command line:

```
./src/index.ts → dist/vue-simpleicons.esm.js...                                                                                                                        
> 1 | import Vue from 'vue';                                                                                                                                           
    |       ^ Unexpected token: name «Vue», expected: punc «;»                                                                                                         
  2 | import Component from 'vue-class-component';                                                                                                                     
  3 | import { Prop } from 'vue-property-decorator';                                                                                                                   
  4 |                                                                                                                                                                  
(!) Unresolved dependencies                                                                                                                                            
https://rollupjs.org/guide/en#warning-treating-module-as-external-dependency                                                                                           
vue (imported by src\VueSimpleIcon.ts)                                                                                                                                 
vue-class-component (imported by src\VueSimpleIcon.ts)                                                                                                                 
vue-property-decorator (imported by src\VueSimpleIcon.ts)                                                                                                              
[!] (plugin uglify) Error: Unexpected token: name «Vue», expected: punc «;»                                                                                            
SyntaxError: Unexpected token: name «Vue», expected: punc «;»                                                                                                          
    at JS_Parse_Error.get (eval at <anonymous> (D:\workspace\vue-simpleicons\node_modules\uglify-js\tools\node.js:20:1), <anonymous>:71:23)                            
    at reportError (D:\workspace\vue-simpleicons\node_modules\jest-worker\build\workers\processChild.js:107:11)                                                        
    at reportClientError (D:\workspace\vue-simpleicons\node_modules\jest-worker\build\workers\processChild.js:87:10)                                                   
    at execFunction (D:\workspace\vue-simpleicons\node_modules\jest-worker\build\workers\processChild.js:157:5)                                                        
    at execHelper (D:\workspace\vue-simpleicons\node_modules\jest-worker\build\workers\processChild.js:139:5)                                                          
    at execMethod (D:\workspace\vue-simpleicons\node_modules\jest-worker\build\workers\processChild.js:143:5)                                                          
    at process.on.request (D:\workspace\vue-simpleicons\node_modules\jest-worker\build\workers\processChild.js:64:7)                                                   
    at process.emit (events.js:198:13)                                                                                                                                 
    at emit (internal/child_process.js:832:12)                                                                                                                         
    at process._tickCallback (internal/process/next_tick.js:63:19)                                                                                                     
                                                                                                                                                                       
error Command failed with exit code 1.                                                                                                                                 
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.                                                                                   
error Command failed with exit code 1.                                                                                                                                 
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.                                                                                   
```